### PR TITLE
Better README

### PR DESCRIPTION
**Pull request by pinebit**
Collected all gained knowledge and re-structured the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,139 @@
...
+# Kurtosis-Charon
+
+## Overview
+
+This project leverages [kurtosis](https://docs.kurtosis.com) to run a local setup of ethereum and beacon chains with DVT.
+It does not connect to real beacon chains and instead it creates a preconfigured local environment, similar to [Ganache](https://archive.trufflesuite.com/ganache/).
+Once started, you will have several instances of execution clients, consensus clients, charon and validator clients running altogether.
+What's important to us: kurtosis takes care of configuring and activating validator keys, we only need to import the keys to run DVT.
+
+Kurtosis supports all of the existing vendors: lighthouse, nimbus, prysm, teku and lodestar for both BN and VC roles.
```
Comment (gsora) [inline README.md:10]: We should link that kurtosis table that details compatibility between BNs and VCs. Not all combinations are supported under kurtosis, even without DVT.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,139 @@
...
 make geth-lighthouse
+make geth-lodestar
+make geth-teku
+make geth-nimbus
+make geth-prysm
 ```
 
-Step 2: Verify that the setup is running in your docker-desktop and wait for 10 seconds. Create a Charon setup by running the following command:
-    
+> Note: this command often fails if you have leftovers from the previous run. In that case, run `make clean` and try again.
```
Comment (gsora) [inline README.md:51]: Might also be worth saying that sometimes docker network-related errors can only be fixed with a complete docker daemon restart.

**Review by gsora — CHANGES_REQUESTED** [review]

**Review by KaloyanTanev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pinebit) [reply README.md:10]: Added

File: README.md (hunk 2 above)
Comment (pinebit) [reply README.md:52]: Added.

File: README.md [hunk 3]
```diff
@@ -1,71 +1,141 @@
...
+You need to have docker installed on your machine, because everything is running in docker.
 
-MAKE SURE `kurtosis-cli` IS THE LATEST VERSION.
+Install `kurtosis-cli` by following the instructions [here](https://docs.kurtosis.com/install).
+This project updates frequently, make sure to update it to the latest version before you start.
 
+On MacOS, simply use `brew`:
+    
 ```shell
+brew install kurtosis-cli
```
Comment (KaloyanTanev) [inline README.md:24]: should be

```suggestion
brew install kurtosis-tech/tap/kurtosis-cli 
```

File: README.md [hunk 4]
```diff
@@ -1,71 +1,141 @@
...
+* CL containers: cl-1-teku-geth...cl-3-teku-geth
+* VC containers: vc-1-teku-geth...vc-3-teku-geth
+
+By default it is running 3 instances of each type. You can change the number of instances: `count: 3` in .yaml files.
+Now you have a local setup of ethereum and beacon chains running and progressing with normal (non-DV) validators.
+
+5. Create charon solo cluster configuration by running:
+
 ```shell
 make charon
```
Comment (KaloyanTanev) [inline README.md:66]: I had to install jq before running this: `brew install jq`.

Comment (KaloyanTanev) [reply README.md:66]: Also the `PROMETHEUS_REMOTE_WRITE_TOKEN` had to be set before running the command, not at step 6.

File: README.md [hunk 5]
```diff
@@ -1,71 +1,141 @@
...
+Also, each charon instance will have its own VC instance of the selected type (vendor).
+This way we create a complete solo DV cluster.
+
+8. Monitoring
+
+If everything runs fine, you will see the DV running and pushing metrics to the Grafana:
+https://grafana.monitoring.gcp.obol.tech/d/b962e704-2e37-48a4-82c0-b15d7661e8a6/charon-overview-v3-testnet-updates?orgId=1&var-cluster_network=testnet
 
-To tear down the project, run the following command:
+> Note that this is the special dashboard designed to monitor "testnet", don't forget to switch to this "Cluster Network". Then select the "Cluster Hash" matching your solo cluster hash found in `.charon/node0/cluster_lock.json`.
```
Comment (KaloyanTanev) [inline README.md:99]:
```suggestion
> Note that this is the special dashboard designed to monitor "testnet", don't forget to switch to this "Cluster Network". Then select the "Cluster Hash" matching your solo cluster hash found in `.charon/cluster/node0/cluster-lock.json`.
```

Comment (pinebit) [reply README.md]: Fixed.

**Pull request merged**